src/it/test-studio/build.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<project name="studio"
```

Comment:
`name=test-studio` ; it's easier for later finding the failing build when named accordingly with the folder/module